niche_vlaanderen/system_tables/niche_vegetation.csv

```diff
@@ -1,1207 +1,1207 @@
-21,Cynosurion cristati,KV,3,2,70,-14,152,33,3,0
-21,Cynosurion cristati,L1,2,1,70,-14,152,33,2,0
-21,Cynosurion cristati,L1,2,1,70,-14,152,33,3,0
-21,Cynosurion cristati,L1,2,2,70,-14,152,33,2,0
-21,Cynosurion cristati,L1,2,2,70,-14,152,33,3,0
-21,Cynosurion cristati,L1,2,3,70,-14,152,33,2,0
-21,Cynosurion cristati,L1,2,3,70,-14,152,33,3,0
-21,Cynosurion cristati,L1,3,1,70,-14,152,33,2,0
-21,Cynosurion cristati,L1,3,1,70,-14,152,33,3,0
-21,Cynosurion cristati,L1,3,2,70,-14,152,33,2,0
-21,Cynosurion cristati,L1,3,2,70,-14,152,33,3,0
-21,Cynosurion cristati,L1,3,3,70,-14,152,33,2,0
-21,Cynosurion cristati,L1,3,3,70,-14,152,33,3,0
-21,Cynosurion cristati,Z1,2,1,80,13,152,89,2,0
-21,Cynosurion cristati,Z1,2,1,80,13,152,89,3,0
-21,Cynosurion cristati,Z1,2,2,80,13,152,89,2,0
-21,Cynosurion cristati,Z1,2,2,80,13,152,89,3,0
-21,Cynosurion cristati,Z1,3,1,80,13,152,89,2,0
-21,Cynosurion cristati,Z1,3,1,80,13,152,89,3,0
-21,Cynosurion cristati,Z1,3,2,80,13,152,89,2,0
-21,Cynosurion cristati,Z1,3,2,80,13,152,89,3,0
-22,Ericion tetralicis,Z1,1,1,47,-20,166,1,1,0
-22,Ericion tetralicis,Z1,1,1,47,-20,166,1,1,2
-22,Ericion tetralicis,Z2,1,1,46,-3,117,36,1,0
-22,Ericion tetralicis,Z2,1,1,46,-3,117,36,1,2
-23,Venige heide,V,1,1,20,-6,74,13,1,0
-23,Venige heide,V,1,2,20,-6,74,13,1,0
-23,Venige heide,Z1,1,1,20,1,74,29,1,0
-23,Venige heide,Z2,1,1,20,1,74,29,1,0
-23,Venige heide,ZV,1,1,20,-6,74,13,1,0
-23,Venige heide,ZV,1,2,20,-6,74,13,1,0
-24,Oxycocco-Ericion,V,1,1,13,-17,43,-8,0,0
-24,Oxycocco-Ericion,V,1,2,13,-17,43,-8,0,0
-24,Oxycocco-Ericion,Z1,1,1,16,-11,23,7,0,0
-24,Oxycocco-Ericion,Z2,1,1,16,-11,23,11,0,0
-24,Oxycocco-Ericion,ZV,1,1,13,-17,3,-8,0,0
-24,Oxycocco-Ericion,ZV,1,2,13,-17,3,-8,0,0
-25,Rynchosporion albae,Z1,1,1,-2,-44,66,-12,0,0
-25,Rynchosporion albae,Z1,1,1,-2,-44,66,-12,0,2
-25,Rynchosporion albae,Z1,1,1,-2,-44,66,-12,1,0
-25,Rynchosporion albae,Z1,1,1,-2,-44,66,-12,1,2
-26,RG Molinia caerulea [Oxycocco-sphagnetea],Z1,2,1,100,-9,127,7,1,0
-26,RG Molinia caerulea [Oxycocco-sphagnetea],Z1,3,1,100,-9,127,7,1,0
-26,RG Molinia caerulea [Oxycocco-sphagnetea],Z2,2,1,100,-9,127,7,1,0
-26,RG Molinia caerulea [Oxycocco-sphagnetea],Z2,3,1,100,-9,127,7,1,0
-26,RG Molinia caerulea [Oxycocco-sphagnetea],ZV,2,1,5,-9,67,7,1,0
-26,RG Molinia caerulea [Oxycocco-sphagnetea],ZV,2,2,5,-9,67,7,1,0
-26,RG Molinia caerulea [Oxycocco-sphagnetea],ZV,3,1,5,-9,67,7,1,0
-26,RG Molinia caerulea [Oxycocco-sphagnetea],ZV,3,2,5,-9,67,7,1,0
-27,RG Myrica gale [Ericion tetralicis],V,1,1,60,-16,126,2,1,0
-27,RG Myrica gale [Ericion tetralicis],V,1,2,60,-16,126,2,1,0
-27,RG Myrica gale [Ericion tetralicis],Z1,1,1,76,-10,126,3,1,0
-27,RG Myrica gale [Ericion tetralicis],Z2,1,1,76,-2,126,33,1,0
-27,RG Myrica gale [Ericion tetralicis],ZV,1,1,22,-3,126,9,1,0
-27,RG Myrica gale [Ericion tetralicis],ZV,1,2,22,-3,126,9,1,0
-28,Calluno-Genistion pilosae,Z1,1,1,172,51,196,114,1,0
+veg_code,veg_type,soil_name,nutrient_level,acidity,mhw_max,mhw_min,mlw_max,mlw_min,management,inundation
```

Comment:
side note (but maybe already handled): I see that you changed the column names. Note that the dataset on zenodo still uses the original names (but with reverted values), see https://zenodo.org/records/10521548 versus the [original niche_vegetation table in v1.x ](https://github.com/inbo/niche_vlaanderen/blob/master/niche_vlaanderen/system_tables/niche_vegetation.csv) versus this version

Reply:
You are correct. We have 2 options to streamline this:
- update the file on Zenodo (this is the easiest way)
- revert the mxw_min and mxw_max column names in the two affected csv files, but then we (at least) need to update the functions related to the checks (Niche._check_all_lower(), Niche._check_input_files()), and other issues with calculation logic. For example: "Vegetation.calculate_deviation()" contains lines in [this block](https://github.com/inbo/niche_vlaanderen/blob/2f85c7da204624a6dec22e19c65ddc06ba7484d0/niche_vlaanderen/vegetation.py#L316) that contains differences and comparisons; this breaks if the column names are reverted. This effect happens in other places and was the reason to change the column names in the first place.

Reply:
Reflecting a bit more about this:

> * update the file on Zenodo (this is the easiest way)

Unfortunately I am a bit reluctant to change this file because it is also used outside `niche_vlaanderen` (a.o: consultancy companies use it in environmental impact assessments, a slightly different version is also included in the official guidelines for environmental impact assessment of groundwater abstraction in the vicinity of Natura 2000 sites). @DriesAdriaens: your thoughts on this?

Reply:
The column name change (and according changes in vegetation.py) has been reverted in revision https://github.com/inbo/niche_vlaanderen/commit/279921faddc31ecbed98a282673a9cd8bd6ab976.